ecommerce/products/base.py

```diff
@@ -0,0 +1,44 @@
+from django.db import models
+
+
+class Rarity:
+    COMMON = 'common'
+    UNCOMMON = 'uncommon'
+    RARE = 'rare'
+    ULTRA_RARE = 'ultra rare'
+    AMAZING_RARE = 'amazing rare'
+    SECRET_RARE = 'secret rare'
```

Comment:
This is good but you want to add a bit called choices which basically makes all of the above attributes available as a list:
```
CHOICES = [
    (COMMON, 'Common'),
    (UNCOMMON, 'Uncommon'),
    ...etc
]
```
The value on the left is the actual value that gets saved in the database and the value on the right is the display value, i.e. what the user sees when selecting a rarity using a form. This will make more sense as we go :)

Reply:
Repeat this pattern for each of these choice classes